tools/contrib/ssp_anomaly_forcing_smooth

```diff
@@ -41,14 +41,21 @@ parser.add_argument(
     action="store_true",
     default=False,
 )
+parser.add_argument(
+    "--output_dir", "--output-dir",
+    help="Top-level output directory (default: ./anomaly_forcing/). Sub-directory will be created for the selected scenario.",
+    type=str,
+    default=os.path.join(".", "anomaly_forcing"),
+)
+
```

Comment:
this anomaly_forcing directory wasn't appended to my output-dir.  should it hav been?

Reply:
`./anomaly_forcing` is just the default. I didn't want to append that automatically to whatever is specified as `output_dir` because I figured some people might want to name their directory something else.